demos/common.scss

```diff
@@ -79,7 +79,6 @@ fieldset {
 
 .catalog-logo,
 .catalog-back {
-  padding-right: 24px;
```

Comment:
We should keep the `padding-right` property for the catalog-logo class.

Reply:
I don't think we should... There is too much space between the logo and the title...

Reply:
Ah wait, there is already a `padding-right:24px;` applied to the remaining class

Reply:
Yup, not needed anymore.

Reply:
That's why I removed it :smiley:

Reply:
Yup, no longer needed.